romancal/regtest/test_ramp_fitting.py

```diff
@@ -188,7 +187,6 @@ def test_rampfit_step(rampfit_result, rtdata_module, ignore_asdf_paths):
     except AssertionError as e:
         error_msgs.append(str(e))
 
-    @metrics_logger(requirement)
     def test_success():
```

Comment:
I don't see the [test_rampfit_step tests](https://github.com/spacetelescope/romancal/pull/1399/files#diff-93200b7e3340cda431dcc72135bd8245d14baa05e4732acbfc6aeb8ec0a00f70L103-L121) for requirements DMS362, DMS366, DMS363, and DMS367 in the fixture above specified in the dms_requirement_tests.json
 below.

I think if you put them under test_rampfit_step in the json, your assert will still work properly as it matches the regex.